src/pyuvsim/simsetup.py

```diff
@@ -2286,11 +2378,17 @@ def initialize_uvdata_from_keywords(
         "array_layout": array_layout,
     }
 
+    ordering_params = {
+        "conjugation_convention": conjugation_convention,
+        "blt_order": blt_order,
+    }
+
     freq_params = {k: v for k, v in freq_params.items() if v is not None}
     time_params = {k: v for k, v in time_params.items() if v is not None}
     selection_params = {k: v for k, v in selection_params.items() if v is not None}
     tele_params = {k: v for k, v in tele_params.items() if v is not None}
     layout_params = {k: v for k, v in layout_params.items() if v is not None}
+    ordering_params = {k: v for k, v in ordering_params.items() if v is not None}
 
     uv_obj = UVData()
```

Comment:
Is this particular instance of a UVdata object _only_ used in 2395? If so maybe we can just put the initializer into that list comprehension? I personally got a little confused why were were creating a blank object instead of using new, util i realized it was just for the extra_kwds dict below.